tags/environmentalsustainability.md

```diff
@@ -0,0 +1,77 @@
+# CNCF TAG Environmental Sustainability
+
+## Introduction
+
+The Paris Climate Accords outline the link between economic activity, greenhouse gas (GHG) emissions, and the impacts of climate change. GHG  emissions, primarily made from Carbon Dioxide, are released during the combustion of fossil fuels to produce electricity. To reach net zero goals, we need to identify how to reduce the environmental footprints of increasingly large data centers.
+
+## Mission Statement
+This TAG's goal is to advocate for, develop, support, and help evaluate environmental sustainability initiatives in Cloud Native Technologies.
+
+All activities of this TAG should aim to positively impact the community by changing mindset, increasing resource efficiency, and putting energy efficiency and environmental sustainability on everyone's agenda.
+
+## Responsibilities & Deliverables
+
+### Background
+We recognize that: 
+- balancing resource consumption with the desire for performance is difficult
+- there is inconsistency in how emissions are calculated and what is included by software in scopes of emission. 
+
+### In-Scope
+
+- Identify, define, and develop tooling to assess and improve environmental sustainability approaches, including
+    - quantification of energy consumption of full-stack software including application, cloud native services
+    - recommendations to manage full-stack software such that the overall energy consumption and associated carbon are reduced.
+    - items to evaluate technological and architectural health of projects
+- Community outreach on any items within scope, for instance, social media/conferences/podcasts/meetups et cetera
+- Collaboration with other sustainability efforts
+
+### Out of scope
+- Form an umbrella organization beyond the CNCF
+- Establish a compliance and standards body beyond the CNCF space
+- Evaluate individual company infrastructures
+- Focus outside of Cloud Native Technologies, according to the [CNCF Cloud Native definition](https://github.com/cncf/toc/blob/main/DEFINITION.md)
+
+## Deliverables to ToC
+- landscape for the carbon and energy metrics/measurements/management savings
```

Comment:
what is a "management savings"?

Reply:
I think we got slash happy.  Looking at this maybe?
landscape for the carbon and energy metrics, measurements, and management

Reply:
Do you mean measurement tools/mechanisms or the real measured data?

Reply:
Both, hence the "carbon and energy metrics, measurements, and management."  Do you have something you would like added here @cathyhongzhang ?

Reply:
"Landscape for carbon and energy efficiency in the form of metrics, measurements, and management techniques"

Reply:
Will add to the PR.